src/components/Verse/VerseText.tsx

```diff
@@ -30,10 +33,16 @@ const VerseText = ({ words, isReadingMode = false }: VerseTextProps) => {
     isReadingMode &&
     (quranTextFontScale > 3 || showWordByWordTranslation || showWordByWordTransliteration);
 
+  const { chapterId } = firstWordData;
```

Comment:
Can we move `isFirstWordOfSurah` here as well instead of using `firstWordData.isFirstWordOfSurah`?